api/routes/user.js

```diff
@@ -53,35 +54,16 @@ router.post('/checkIfUserExists', (req, res) => {
 });
 
 // Register a member
-router.post('/register', function(req, res) {
-  if (req.body.email && req.body.password) {
-    const newUser = new User({
-      password: req.body.password,
-      firstName: req.body.firstName,
-      middleInitial: req.body.middleInitial || '',
-      lastName: req.body.lastName,
-      email: req.body.email.toLowerCase(),
-      major: req.body.major || ''
-    });
-
-    const membershipValidUntil = getMemberValidationDate(
-      req.body.numberOfSemestersToSignUpFor
-    );
-    newUser.membershipValidUntil = membershipValidUntil;
-
-    const testPassword = testPasswordStrength(req.body.password);
-
-    if (!testPassword.success) {
-      return res.status(BAD_REQUEST).send({ message: testPassword.message });
+router.post('/register', async (req, res) => {
+  const regRes = await registerUser(req.body);
```

Comment:
Name this something better, e.g.
```suggestion
  const registrationStatus = await registerUser(req.body);
```